addons/mail/static/src/models/messaging_notification_handler.js

```diff
@@ -47,8 +47,11 @@ registerModel({
             const proms = notifications.map(message => {
                 if (typeof message === 'object') {
                     switch (message.type) {
-                        case 'bus/im_status':
-                            return this._handleNotificationBusImStatus(message.payload);
+                        case 'record/insert':
+                            for (const model in message.payload) {
+                                this.messaging.models[model].insert(message.payload[model]);
+                            }
```

Comment:
model manager should have a method to insert all records in a single update cycle, taking as param the newly created format.

Old insert should be deprecated and eventually not used, even in JS.